package.json

```diff
@@ -36,7 +36,7 @@
     "ember-cli-typescript": "2.0.0-beta.3",
     "fs-extra": "7.0.0",
     "json-typescript": "1.0.1",
-    "qunit-snapshot": "1.2.1",
+    "qunit-snapshot": "1.3.0",
```

Comment:
Any reason to lock this down (and prevent bug fixes from trickling down to users)?

Reply:
Still experimenting with @renovate-bot -- this is what it does by default

Reply:
Fixed 4b6860f